vm/copy_test.go

```diff
@@ -0,0 +1,28 @@
+package vm
+
+import (
+	"testing"
+)
+
+func TestCopy(t *testing.T) {
+	env := NewEnv()
+	env.Define("a", "a")
+	copy := env.Copy()
+	if v, e := copy.Get("a"); e != nil || v != "a" {
+		t.Errorf("Copy doesn't retain original values")
```

Comment:
please use lower case string in first word if it is not identifier. and below.

Reply:
Done